addons/hr_recruitment/hr_recruitment_view.xml

```diff
@@ -109,6 +109,8 @@
                 <div class="oe_title">
                     <label for="name" class="oe_edit_only"/>
                     <h1><field name="name"/></h1>
+                    <label for="categ_ids"/>
+                    <field name="categ_ids" widget="many2many_tags"/>
```

Comment:
either use "oe_edit_only" class on label for category ids or use placeholder without label.